spec.bs

```diff
@@ -2359,11 +2394,11 @@ of the following global objects:
     1. Let |browserSignalsJS| be |browserSignals| [=converted to ECMAScript values=].
     1. Let |directFromSellerSignalsJs| be |directFromSellerSignalsForBuyer|
       [=converted to ECMAScript values=].
-    1. Let |startTime| be the [=current wall time=].
+    1. Let |startTime| be the [=environment settings object/current monotonic time=].

```

Comment:
You need to get this from an actual ESO variable or global. This algorithm doesn't take one in, so you can either keep using `[=current wall time=]` or plumb an ESO in. I'm not sure using an ESO's current time is correct here anyways, since the global is specific to a document and this algorithm is running in parallel (not to mention that the Document might be destroyed by this time anyways).

Reply:
I'll be honest, I don't know the difference between the two to have a recommendation. I think monotonic makes sense; I'm really not sure why it's tied to an ESO though, if the spec says it is a UA-global system-clock primitive.

> And this algorithm does create a realm and get its global at the first two steps. Can we use its ESO to get the monotonic time?

Ah, yes you can get an ESO from the current script runner. Note that technically right now you can't, but once I fix https://github.com/WICG/turtledove/issues/676 then you will be able to. You can grab it and put an XXX box with an `Issue: ` thing in bikeshed pointing to https://github.com/WICG/turtledove/issues/676.

Reply:
Ah thanks! Done.

Reply:
Thanks, I'll mark this as resolved.